app/components/Incentive.scss

```diff
@@ -20,6 +20,12 @@
         }
     }
 }
+.time{
+  margin-left: 35px;
+}
+.goto{
+  margin-left: 90px;
```

Comment:
Do not apply this style on small screens because it looks broken:
![image](https://user-images.githubusercontent.com/1814312/29666875-7c545cea-88da-11e7-812b-27a1da79a262.png)